### DISCUSSION TITLE
How can I modify the navbar on mobile screen ?

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [fuma-nama/fumadocs#966] [Q&A] [answered]
I want to add this section which is below the toc to the navbar ? Is there any way to do that ? 

![image](https://github.com/user-attachments/assets/975f2491-6077-4596-b862-76f4d28825e9)
The `change appearance` button beside the search icon on mobile screen 

![image](https://github.com/user-attachments/assets/829a61a6-db77-4ff5-bd54-47a0ee78a08a)

Maybe like this 

![image](https://github.com/user-attachments/assets/e41a84da-2697-4240-b206-f21d3c5735fe)

Is there a way I can just add somethings on the mobile and not on the navbar.

**Comment by fuma-nama** [comment, marked as answer]
You can replace the default navbar with your custom one from docs layout
